source/javascripts/components/workflow-recipes/WorkflowRecipeLink/WorkflowRecipeLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Flex, Icon, Link, Text, Tooltip } from "@bitrise/bitkit";
+
+type WorkflowRecipeLinkProps = {
+	id: string;
+	hideIcon?: boolean;
+}
+
+const WorkflowRecipeLink = ({ id, hideIcon }: WorkflowRecipeLinkProps): JSX.Element => (
```

Comment:
Just nitpicking, but what do you think about naming this component `WorkflowRecipesLink`